lookerpy/configuration.py

```diff
@@ -55,16 +67,28 @@ def __init__(self):
         """
         Constructor
         """
+        self.configuration_yaml = 'config.yml'
+        self.environment = 'production'
+
+        configuration_yaml = self.configuration_yaml
+        environment = self.environment
+
+        self.client_id, self.client_secret = _fetch_credentials(configuration_yaml, environment)
         # Default Base url
-        self.host = "https://looker-stage-vpc.knewton.net:29999/api/3.0"
+        self.host = "https://looker.knewton.net:29999/api/3.0"
```

Comment:
Can we make this part of the config.yml so that we can move between staging and prod easily?